pkg/provider/managedk8s/ruleset/securityhardenedk8s/rules/2000.go

```diff
@@ -55,6 +55,11 @@ func (r *Rule2000) Severity() rule.SeverityLevel {
 }
 
 func (r *Rule2000) Run(ctx context.Context) (rule.RuleResult, error) {
+	const (
+		namespaceDeletionWithoutPodsMessage = "namespace is marked for deletion - no pods are deployed on it"
+		namespaceDeletionWithPodsMessage    = "namespace is marked for deletion - there are still pods deployed on it"
```

Comment:
```suggestion
		namespaceDeletionWithoutPodsDetails = "namespace is marked for deletion with no present pods"
		namespaceDeletionWithPodsDetails    = "namespace is marked for deletion with present pods"
		ingressTrafficNotDeniedMessage = "Ingress traffic is not denied by default."
		egressTrafficNotDeniedMessage = "Egress traffic is not denied by default."
```

The "not denied by default" message is repeated several times, we can store it in a const.